client/src/Booking.jsx

```diff
@@ -42,14 +42,13 @@ class Booking extends React.Component {
 
   componentDidMount() {
     this.init();
-    this.book();
-    this.bookingTotal(); //testing
+    this.bookingTotal();
   }
 
   init() {
     axios.get('http://localhost:3002/booking')
     .then(({data}) => {
-      console.log(data)
+      console.log('data: ', data)
```

Comment:
i see this call being successful on my server side, but none of the date renders on the webpage when i run you front end. 
![Screen Shot 2021-05-29 at 10 14 25 AM](https://user-images.githubusercontent.com/71669711/120078943-a6a59a00-c066-11eb-9c42-9cb85b7a45cc.png)